cms/models.py

```diff
@@ -736,3 +740,195 @@ class ResourcePagesSettings(BaseSetting):
 
     class Meta:
         verbose_name = "Resource Pages"
+
+
+class SignatoryObjectIndexPage(Page):
+    """
+    A placeholder class to group signatory object pages as children.
+    This class logically acts as no more than a "folder" to organize
+    pages and add parent slug segment to the page url.
+    """
+
+    class Meta:
+        abstract = True
+
+    parent_page_types = ["HomePage"]
+    subpage_types = ["SignatoryPage"]
+
+    @classmethod
+    def can_create_at(cls, parent):
+        """
+        You can only create one of these pages under the home page.
+        The parent is limited via the `parent_page_type` list.
+        """
+        return (
+            super().can_create_at(parent)
+            and not parent.get_children().type(cls).exists()
+        )
+
+    def serve(self, request, *args, **kwargs):
+        """
+        For index pages we raise a 404 because these pages do not have a template
+        of their own and we do not expect a page to available at their slug.
+        """
+        raise Http404
+
+
+class SignatoryIndexPage(SignatoryObjectIndexPage):
+    """
+    A placeholder page to group all the signatories under it as well
+    as consequently add /signatories/ to the signatory page urls
+    """
+
+    slug = SIGNATORY_INDEX_SLUG
+
+
+class SignatoryPage(Page):
+    """ CMS page representing a Signatory. """
+
+    promote_panels = []
+    parent_page_types = [SignatoryIndexPage]
+    subpage_types = []
+
+    name = models.CharField(
+        max_length=250, null=False, blank=False, help_text="Name of the signatory."
+    )
+    title_1 = models.CharField(
+        max_length=250,
+        null=True,
```

Comment:
FYI: It's a good practice to not use `null=True` for Char or Text Fields.
https://simpleisbetterthancomplex.com/tips/2016/07/25/django-tip-8-blank-or-null.html